k2/torch/csrc/fsa_class_test.cu

```diff
@@ -531,8 +507,5 @@ TEST(RaggedArcTest, FromBinaryFunctionTensor) {
 
 int main(int argc, char *argv[]) {
```

Comment:
Please remove `main` and link to the library `gtest_main`.

Please see what we are doing in other `xxx_test.cu` files.